extensions/positron-r/src/provider.ts

```diff
@@ -526,6 +532,26 @@ function discoverAdHocBinaries(paths: string[]): RBinary[] {
 		.map(b => ({ path: b, reasons: [ReasonDiscovered.adHoc] }));
 }
 
+/**
+ * Discovers R binaries that are relevant to Posit Workbench.
+ * Paths are from: https://docs.posit.co/ide/server-pro/r/using_multiple_versions_of_r.html
+ * @returns R binaries that are relevant to Posit Workbench.
+ */
+function discoverPWBBinaries(): RBinary[] {
+	const pwbBinaries = discoverHQBinaries([
+		'/usr/lib/R',
+		'/usr/lib64/R',
+		'/usr/local/lib/R',
+		'/usr/local/lib64/R',
+		'/opt/local/lib/R',
+		'/opt/local/lib64/R',
+		// '/opt/R', // Already checked for in rHeadquarters
+		'/opt/local/R'
+	]);
```

Comment:
@melissa-barca this PR adds handling to locate installations of R under these directories as long as they are in a subdirectory with format `<R_VERSION>/bin/R`.

Is this consistent with R installation paths on Workbench, or do we expect that some admins have more complex paths to R?

Reply:
I am wondering same, i.e. is this definitely prepped for real-world situations on linux? I think I'm mostly wondering if all of these should be treated at root or HQ folders or if some of them are more likely to be a symlink to one specific R binary.

Reply:
RStudio looks at any subdirectories within these directories for `bin/R`: https://github.com/rstudio/rstudio/blob/80d22eb99fd4a6fdc5687473f0b9dc776cd85e2f/src/cpp/core/r_util/RVersionsPosix.cpp#L147

Reply:
@jennybc maybe we want to consider these `ReasonDiscovered.HQ` after all? Or do we want to distinguish these paths from our existing HQ interpreters?

Melissa's comment from https://github.com/posit-dev/positron/pull/6272#discussion_r1951323753
> PWB checks these paths because they are default linux installation locations or best practice install locations, none of them are necessarily Workbench-specific or exclusive so even though that is our primary intent for including them, we might want more generic messaging here.

Reply:
I don't have a super strong opinion. For context, in terms of determining the somewhat-nebulous concept of "HQ for this OS", I consult https://rstudio.github.io/r-manuals/r-admin/Installing-R-under-Unix-alikes.html#installation (that would be the OS relevant to this discussion).

Reply:
And that resource says that

> ... `/usr/local/bin` (the default place for installation of R)